### PR TITLE
Fix go install v2

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description

go install github.com/googlecloudplatform/gcsfuse@v2.0.0 command gives the following error:

```
go: github.com/googlecloudplatform/gcsfuse@v2.0.0: github.com/googlecloudplatform/gcsfuse@v2.0.0: invalid version: module contains a go.mod file, so module path must match major version ("github.com/googlecloudplatform/gcsfuse/v2")
```

This used to work till version 1.4.2
Fix
8ywradKzo2qfHNt

We have to use 
```
go install github.com/googlecloudplatform/gcsfuse/v2@v2.0.0
```

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Tested on forked repo, installation of v2 as well v1 also working, tested on louhi
2. Unit tests - NA
3. Integration tests - Automated

### CODE REVIEW

**Comment by raj-prince** [comment]
Changes look good to me.

Just wanted to confirm a couple of things:
(a) `go install github.com/googlecloudplatform/gcsfuse@master` - will install the latest version (v2) or old version (v1)?

(b) `go install github.com/googlecloudplatform/gcsfuse@master` will be same as `go install github.com/googlecloudplatform/gcsfuse/v2@master`?

File: tools/util/build_gcsfuse.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func BuildGcsfuse(dstDir string) (err error) {
 	{
 		var pkg *build.Package
 		pkg, err = build.Import(
-			"github.com/googlecloudplatform/gcsfuse",
+			"github.com/googlecloudplatform/gcsfuse/v2",
```
[the remark this answers is not in the document]
Comment (Tulsishah) [reply tools/util/build_gcsfuse.go:56]: Yes for building package.

**Review by ashmeenkaur — APPROVED** [review]

**Comment by Tulsishah** [comment]
> Changes look good to me.
> 
> Just wanted to confirm a couple of things: (a) `go install github.com/googlecloudplatform/gcsfuse@master` - will install the latest version (v2) or old version (v1)?
According to my testing it will install latest commit before this change.

> (b) `go install github.com/googlecloudplatform/gcsfuse@master` will be same as `go install github.com/googlecloudplatform/gcsfuse/v2@master`?

`go install github.com/googlecloudplatform/gcsfuse/v2@master` --> This will pick latest commit of master after this change.

Although I raised yaqs question regarding to this. Will confirm you If I get any response.

**Comment by raj-prince** [comment]
> > Changes look good to me.
> > Just wanted to confirm a couple of things: (a) `go install github.com/googlecloudplatform/gcsfuse@master` - will install the latest version (v2) or old version (v1)?
> 
> According to my testing it will install latest commit before this change.
> 
> > (b) `go install github.com/googlecloudplatform/gcsfuse@master` will be same as `go install github.com/googlecloudplatform/gcsfuse/v2@master`?
> 
> `go install github.com/googlecloudplatform/gcsfuse/v2@master` --> This will pick latest commit of master after this change.
> 
> Although I raised yaqs question regarding to this. Will confirm you If I get any response.


> [princer] I am asking that because - we need to update our public documentation accordingly - https://cloud.google.com/storage/docs/gcsfuse-install#install-source-code

> [princer] Also, if you may do a quick POC by creating a simple go module and releasing version v1 (prints version v1) and v2 (prints v2) on that.

**Comment by Tulsishah** [comment]
> [princer] I am asking that because - we need to update our public documentation accordingly - https://cloud.google.com/storage/docs/gcsfuse-install#install-source-code
 Yes got it
> [princer] Also, if you may do a quick POC by creating a simple go module and releasing version v1 (prints version v1) and v2 (prints v2) on that. 

I push v1 tag and then push v2 tag, and observe this 
image

**Comment by raj-prince** [comment]
> > [princer] I am asking that because - we need to update our public documentation accordingly - https://cloud.google.com/storage/docs/gcsfuse-install#install-source-code
> > Yes got it
> > [princer] Also, if you may do a quick POC by creating a simple go module and releasing version v1 (prints version v1) and v2 (prints v2) on that.
> 
> I push v1 tag and then push v2 tag, and observe this image



> > [princer] I am asking that because - we need to update our public documentation accordingly - https://cloud.google.com/storage/docs/gcsfuse-install#install-source-code
> > Yes got it
> > [princer] Also, if you may do a quick POC by creating a simple go module and releasing version v1 (prints version v1) and v2 (prints v2) on that.
> 
> I push v1 tag and then push v2 tag, and observe this image

This is inline with my understanding. Then as an action item, we need to update our GCSFuse installation doc

**Review by raj-prince — APPROVED** [review]

**Pull request merged**